pkg/stub/imagestreams.go

```diff
@@ -384,7 +384,7 @@ func (h *Handler) processImportStatus(is *imagev1.ImageStream, cfg *v1.Config) (
 		}
 
 		if !anyErrors {
-			if len(is.Spec.Tags) == len(is.Status.Tags) {
+			if len(is.Spec.Tags) <= len(is.Status.Tags) {
```

Comment:
nit - add a comment to make it clear this covers an upgrade where an upstream tag was removed.

Reply:
comment added @adambkaplan .... figured it was a safe change for me to make given my jetlagged self :-)